workshop_operations/onboarding-checklist.md

```diff
@@ -4,61 +4,84 @@
 ---
 
-* **How do I find co-instructors and helpers?**
-- [ ] Ask the community via organizers @ swcarpentry dot uio dot no
-- [ ] For helpers: ask also for new helpers via announcements @ swcarpentry dot uio dot no
+### **I am looking for co-instructors and helpers!**
+- [ ] Ask the community by sending an email to organizers @ swcarpentry dot uio dot no
+- [ ] Tip: ask for new helpers via announcements @ swcarpentry dot uio dot no
 ---
 
-* **How do I create a workshop website?**
-- [ ] You must have a GitHub profile
-- [ ] Ask organizer @ swcarpentry dot uio dot no to invite you as a member
-- [ ] Go to Carpentry@UiO on GitHub https://github.com/uio-carpentry
-- [ ] Open https://github.com/carpentries/workshop-template in another tab of the web browser, and click "Use this template" button in green
-- [ ] Choose "uio-carpentry" as a owner, set the name of the repository YYYY-MM-DD-lesson, make sure that "Public" is chosen and click "Create repository from template" button in green. 
+### **I want to create a workshop website. How do I go about it?**
+- [ ] Create a GitHub profile for yourself
+- [ ] Ask carpadmin @ swcarpentry dot uio dot no to invite you as a member of uio-carpentry https://github.com/uio-carpentry
+- [ ] Open https://github.com/carpentries/workshop-template and click the ```Use this template``` button in green
+- [ ] Choose "uio-carpentry" as owner, set the name of the repository YYYY-MM-DD-lesson, make sure that "Public" is chosen and click "Create repository from template" button in green.
 - [ ] Edit the necessary parts by following README.md "Customizing Your Website"
 - [ ] The website will be automatically created
-- [ ] Paste the link into the planning spreadsheet
+- [ ] Send the link to the website to carpadmin @ swcarpentry dot uio dot no – the coordinators will create an event on the University Library's course catalogue and create a registration form
+- [ ] Submit your workshop to the Carpentries database by following the instructions on this website https://amy.carpentries.org/forms/self-organised/
 ---
 
-* **How do I get in contact with workshop participants?**
-- [ ] Download registration list from Nettskjema
-- [ ] Send email to those who registered, using BCC
-- [ ] Send reminder before workshop
+### **How do I get in contact with workshop participants?**
+- [ ] Registration for workshops is handled by the local coordinator
+- [ ] By default, registration is closed two weeks prior to the workshop date
+- [ ] Automatic reminders for workshop participants are sent out via the registration system one week and again three days before the workshop
+- [ ] If you want to get in contact with participants, the local coordinator will send you the participants contacts
+- [ ] Workshop participants can ask for a Certificate of Attendance after the workshop; the certificate is generated automatically
 ---
 
-* **Which room(s) will I be teaching in?**
+### **Which room(s) will I be teaching in?**
```

Comment:
Should it be described that a room is most of the time already booked when a workshop calls for co-instructors, and in that case it will be most probably the seminar room 123 in VBH? I am not sure if the answers to this question should be written in the form of checklist.

Reply:
Keep as Annika suggested, as the room may change in future.